HandheldCompanion/Views/Pages/LibraryPage.xaml.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using HandheldCompanion.Controls.Hints;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Timers;
+using System.Windows;
+using Page = System.Windows.Controls.Page;
+using GameLib;
+using HandheldCompanion.ViewModels;
+using HandheldCompanion.Controls;
+using Microsoft.Extensions.DependencyModel;
+using System.Diagnostics;
+
+namespace HandheldCompanion.Views.Pages
+{
+    /// <summary>
+    /// Interaction logic for LibraryPage.xaml
+    /// </summary>
+    public partial class LibraryPage : Page
+    {
+        public delegate void StatusChangedEventHandler(int status);
+        public event StatusChangedEventHandler StatusChanged;
+
+        private Timer timer;
+        private int prevStatus = -1;
+
+        public static LauncherManager _launcherManager { get; set; }
+        public static GameViewModel _gameViewModel { get; set; }
+
+        public LibraryPage()
+        {
+            InitializeComponent();
+            _launcherManager = new LauncherManager();
+            _gameViewModel = new GameViewModel(_launcherManager);
+            DataContext = _gameViewModel;
+            //if (Library.Children.Count == 0)
+            Games.Children.Add(new GameView());
```

Comment:
Do not do this. Avoid modifying structures or anything through back-end code.
All a page should do is have the viewmodel set as DataContext, that's it.

Reply:
So move the Gameview code from that style to library page then just set data context?

Reply:
Since "Games" only has one item, just move over the xaml code from GameView to LibraryPage.
GameView shouldn't exist since in it's current state it is basically your library page.